src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -65,7 +67,8 @@ public static void SetCurrentCircuitHost(CircuitHost circuitHost)
 
             Services = scope.ServiceProvider;
 
-            Circuit = new Circuit(this);
```

Comment:
Does this super need to change? It's intentional that the circuit is **dumb** - I constructed it here to ensure that it stays that way.